tests/deployment_test.go

```diff
@@ -1044,6 +1065,103 @@ func (s *DeploymentSuite) checkListAndWaitForBatchCompletion(ctx context.Context
 	}
 }
 
+func (s *DeploymentSuite) waitForChan(ctx context.Context, ch chan struct{}) {
+	s.T().Helper()
+	select {
+	case <-ch:
+	case <-ctx.Done():
+		s.FailNow("context timeout")
+	}
+}
+
+func (s *DeploymentSuite) TestUpdateWorkflowExecutionOptions_ChildWorkflowWithSDK() {
```

Comment:
I thought we want to use verisoning_3_test for testing versioning behaviors/overrides, etc, and keep deployment_test for deployment APIs.

Reply:
Ah yes, I can move stuff over.